mavlink/definitions/uavionix.xml

```diff
@@ -0,0 +1,399 @@
+<?xml version='1.0'?>
+<mavlink>
+    <include>common.xml</include>
+    <enums>
+      <enum name="UAVIONIX_SDA_LEVEL">
```

Comment:
Ah, I see it in a few places. It just felt super verbose, but obviously doesn't matter.